src/routes/Router.tsx

```diff
@@ -32,14 +38,32 @@ const Router = () => {
         <Route path="/legal/termsAndConditions" element={<Legal document="termsAndConditions" />} />
         <Route path="/legal/privacyPolicy" element={<Legal document="privacyPolicy" />} />
         <Route path="/legal/cookiePolicy" element={<Legal document="cookiePolicy" />} />
+        {renderLegacyRoute(legacyCreateBoard)}
         <Route
-          path="/new"
+          path="/boards"
           element={
             <RequireAuthentication>
-              <NewBoard />
+              <Boards />
             </RequireAuthentication>
           }
-        />
+        >
+          <Route index element={<Navigate to="templates" />} />
+          <Route path="templates" element={<Templates />}>
+            {/* TODO extract settings routes to avoid repetition */}
+            <Route path="settings" element={<SettingsDialog enabledMenuItems={{appearance: true, feedback: feedbackEnabled, profile: true}} />}>
+              <Route path="appearance" element={<Appearance />} />
+              <Route path="feedback" element={<Feedback />} />
+              <Route path="profile" element={<ProfileSettings />} />
+            </Route>
+          </Route>
+          <Route path="sessions" element={<Sessions />}>
+            <Route path="settings" element={<SettingsDialog enabledMenuItems={{appearance: true, feedback: feedbackEnabled, profile: true}} />}>
+              <Route path="appearance" element={<Appearance />} />
+              <Route path="feedback" element={<Feedback />} />
+              <Route path="profile" element={<ProfileSettings />} />
+            </Route>
+          </Route>
+        </Route>
```

Comment:
Should we hide the sessions for anonymous users?

Reply:
personally I think there's nothing wrong with anonymous users being able to join existing sessions.

However, I admit it could be confusing for users who are not technically experienced though, because they will only see a hand full of session at most since it expires after a while. Knowing that, it would make sense to restrict the feature to logged in users as these are kept.

Reply:
will discuss this another time